scripts/src/write_df_from_fastq.R

```diff
@@ -190,6 +202,13 @@ write_df_from_fastq_DRAGEN <- function(
     dplyr::summarise(n = sum(n, na.rm = T)) %>% 
     dplyr::ungroup()
   print (paste("collapsed across", length(cumulative_count_df_uncollapsed$flowcell_name %>% unique()), "flowcells"))
+  
+  # remove index_2 column if it has NAs due to the experiment using single-index barcodes
```

Comment:
So then are we deciding to drop the index_2 column if it is all NAs within Nori itself?
(I am not very opinionated)

Reply:
Yes, I believe this is what we decided at the seq pipeline meeting

Reply:
Yes!